### PR TITLE
Add myself as a speaker

### PR DESCRIPTION
**Pull request by ArmandoLacerda**
Adding myself to the speakers list.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=429) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=429) before we can accept your contribution.<br/><br/>:x: ArmandoLacerda [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=429)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/website?pullRequest=429) it.</sub>

File: input/community/speakers/armando-lacerda.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - Artificial Intelligence
+  - Azure
+  - Big Data
+  - C#
+  - Containers
+  - Data
+  - ML.NET
+  - Serverless
+Blog: www.armandolacerda.com
+Twitter: @arlacerda
```
Comment (daveaglick) [inline input/community/speakers/armando-lacerda.md:19]: It's expecting a full URL here, so https://twitter.com/arlacerda

Comment (SeanKilleen) [inline input/community/speakers/armando-lacerda.md:19]:
```suggestion
Twitter: https://twitter.com/arlacerda
```

**Review by SeanKilleen — COMMENTED** [review]
Suggesting changes to help the build pass.

**Pull request merged**